model/Core/Classes/Entity.md

```diff
@@ -0,0 +1,20 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# Entity
+
+## Summary
+
+TODO
+
+## Description
+
+TODO
+
+## Metadata
+
+- name: Entity
+- SubclassOf: Element
```

Comment:
I just stumbled upon [this](https://github.com/spdx/spdx-3-model/issues/35#issuecomment-1298900696) remark by @iamwillbar stating that Actor (now Entity) should not inherit from Element to avoid cyclic dependencies. Has this been discussed any further?